specification/batch/data-plane/Microsoft.Batch/stable/2024-07-01.20.0/BatchService.json

```diff
@@ -2774,6 +2774,17 @@
               "postfix": "Options"
             }
           },
+          {
+            "name": "forceAfterSeconds",
```

Comment:
I see the on BatchService repo, it has been changed to `force`

Reply:
good catch, updated